.github/workflows/node.js.yml

```diff
@@ -53,6 +53,29 @@ jobs:
                   file: ./coverage/lcov.info
                   flags: codewhisperer
 
+    web:
+        name: test Web
+        runs-on: macos-latest
```

Comment:
Not a blocker, but we should probably do Linux for this case. Because it's faster than macOS. Unless there's a reason not to.

Reply:
Nope no reason for mac. Good to know linux is faster, will updated.